docs/source/platform/schema-validation.md

```diff
@@ -80,90 +123,40 @@ type Query {
-```
-
-> For accuracy, it's best to retrieve the schema from a running GraphQL server (with introspection enabled), though the CLI also reference a local file. See [config options](../platform/apollo-config.html) for more information.
-
-After analyzing the changes against current usage metrics, Apollo will identify three categories of changes and report them to the developer on the command line or within a GitHub pull-request:
-
-1. **Failure**: Either the schema is invalid or the changes _will_ break current clients.
-2. **Warning**: There are potential problems that may come from this change, but no clients are immediately impacted.
-3. **Notice**: This change is safe and will not break current clients.
-
-The more [performance metrics](./performance.html) that Apollo has, the better the report of these changes will become.
-
-<h2 id="github">GitHub Integration</h2>
+Schema validation is best used when integrated in a team's development workflow. To make this easy, Apollo integrates with GitHub to provide status checks on pull requests when schema changes are proposed. To enable schema validation in GitHub, follow these steps:
 
 ![GitHub Status View](../img/schema-history/github-check.png)
 
-Schema validation is best used when integrated in a team's development workflow. To make this easy, Apollo integrates with GitHub to provide status checks on pull requests when schema changes are proposed. To enable schema validation in GitHub, follow these steps:
-
 <h3 id="install-github">Install GitHub application</h3>
 
 Go to [https://github.com/apps/apollo-engine](https://github.com/apps/apollo-engine) and click the `Configure` button to install the Apollo Engine integration on the appropriate GitHub profile or organization.
 
 <h3 id="check-schema-on-ci">Run validation on each commit</h3>
 
-By enabling schema validation in a continuous integration workflow (e.g. CircleCI, etc.), validation can be performed automatically and potential problems can be displayed directly on a pull-request's status checks — providing feedback to developers where they can appreciate it the most.
+After adding `apollo service:check` in a continuous integration workflow (e.g. CircleCI, etc.), schema validation is performed automatically and potential problems are displayed directly on a pull-request's status checks, providing actionable feedback to developers.
+
+To setup validation, run the `apollo service:check` command targeting a GraphQL server with introspection enabled. An example of is shown below with a CircleCI config:
 
-To run the validation command, the GraphQL server must have introspection enabled and run the `apollo service:check` command. An example of what this could look like is shown below with a CircleCI config:
+> Note: with a GitHub status check, ignoring the `apollo`'s error code by appending `|| echo 'validation failed'` allows continuous integration to continue without failing early
```

Comment:
nit: Maybe more instructive here? "To allow continuous integration to continue without failing early, ignore the exit code of the `apollo service:check command`. You can do this by appending `|| echo 'validation failed'` to the command call.